docs/tctl/workflow/run.md

```diff
@@ -104,7 +104,11 @@ tctl workflow run --cron <string>
 Specify a [Workflow Id Reuse Policy](/concepts/what-is-a-workflow-id-reuse-policy).
 Configure if the same [Workflow Id](/concepts/what-is-a-workflow-id) is allowed for use in new [Workflow Execution](/concepts/what-is-a-workflow-execution).
 
-Values: `AllowDuplicate`, `AllowDuplicateFailedOnly`, `RejectDuplicate`
+There are three allowed values:
+
+- **AllowDuplicateFailedOnly:** Allow starting a Workflow execution using the same Workflow Id when a Workflow with the same Workflow Id is not already running and the last execution close state is one of _[terminated, cancelled, timedout, failed]_.
```

Comment:
We can link to the definition of these -

Reply:
Other than the enum page and others, what definition is there to link? There isn't a set thing; these definitions are repeated across pages.

Reply:
They are defined here: https://docs.temporal.io/workflows#workflow-id-reuse-policy

I think we should turn each of them into a header and you can list them, but link to each one directly - or something.
I usually prefer to link to source than repeat that many times.